nucleus/payload_constructor.py

```diff
@@ -39,6 +44,17 @@ def construct_annotation_payload(
     return {ANNOTATIONS_KEY: annotations, ANNOTATION_UPDATE_KEY: update}
 
 
+def construct_segmentation_payload(
+    annotation_items: List[SegmentationAnnotation],
+    update: bool,
+) -> dict:
+    annotations = []
+    for annotation_item in annotation_items:
+        annotations.append(annotation_item.to_payload())
+
+    return {SEGMENTATIONS_KEY: annotations, "force": update}
```

Comment:
Believe we decided to standardize on `update` as the key name for this parameter instead of `force`

Reply:
yup, changed it here now that the switch is deployed on prod